src/app/shared/services/subsigners/walletconnect-subsigner.service.ts

```diff
@@ -47,13 +47,23 @@ export class WalletConnectSubsignerService implements Subsigner {
       map((lib) => {
         this.walletConnectProvider = new lib.default({
           chainId: this.preferenceStore.getValue().chainID,
+          infuraId: "27e484dcd9e3efcfd25a83a78777cdf1",
```

Comment:
Is it safe to publicly provided `infuraId`?

Reply:
Well, I don't ever read from blockchain via this provider because I just use it as a signer. This `infuraId` is some stock ID from their documentation. I left it here because it's a mandatory field. Also, I'm still evaluating this walletconnect implementation as it doesn't work quite well... But thanks for pointing out, I'll extract it to an environment variable.